src/arc.ts

```diff
@@ -132,34 +161,45 @@ export async function initializeArc(provider?: any): Promise<boolean> {
     const readonly = typeof provider === "string";
 
     // if there is no existing arc, we create a new one
-    if ((window as any).arc) {
-      arc = (window as any).arc;
+    if ((window as any).arcs[network]) {
```

Comment:
we can also specify this type globaly like this:
```ts
type IArc = any;
declare global {
    interface Window { arc: IArc; }
}
```
[more details here](https://stackoverflow.com/questions/12709074/how-do-you-explicitly-set-a-new-property-on-window-in-typescript)